assets/js/components/audit-opinions.js

```diff
@@ -20,14 +20,14 @@ class ReportCard {
     this.$element.find(".audit-outcome__year").text(formatFinancialYear(report.date));
     this.$element.find(".audit-outcome__heading").text(report.result);
 
-    if (report.report_url.endsWith('.pdf')) {
+    if (report.report_url === null) {
+      this.$element.find(".audit-outcome__download").text("No report available");
+    }
+    else if (report.report_url.endsWith('.pdf')) {
       report.report_url = report.report_url.substring(0, report.report_url.lastIndexOf("/"));
     }
 
     this.$element.find(".audit-outcome__download").attr("href", report.report_url);
```

Comment:
I think this needs to be included in the "if null" else clause, because it doesn't make sense to set href=null, right?

Reply:
Changed to not set href to null